### PR TITLE
executing a request to the kh endpoint instead of creating a Request object

### PR DESCRIPTION
**Pull request by elebertus**
This addresses https://github.com/kuberhealthy/kuberhealthy/issues/1165

### CODE REVIEW

**Comment by elebertus** [comment]
Rebased for signoff to satisfy the DCO bot.

File: pkg/checks/external/nodeCheck/main.go [hunk 1]
```diff
@@ -85,15 +85,17 @@ func waitForKuberhealthyEndpointReady(ctx context.Context, kuberhealthyEndpoint
 			return doneChan
 		default:
 		}
-
-		_, err := http.NewRequest("GET", kuberhealthyEndpoint, nil)
-		if err == nil {
+		res, err := http.Get(kuberhealthyEndpoint)
+		if err != nil {
+			log.Debugln(kuberhealthyEndpoint, "Error making reuqest to kuberhealthy endpoint "+err.Error())
```
Comment (thatfatguypat) [inline pkg/checks/external/nodeCheck/main.go:90]: You've got a typo here on `reuqest` , but otherwise I think this looks sensible

Comment (elebertus) [reply pkg/checks/external/nodeCheck/main.go]: Got it, thanks!

File: pkg/checks/external/nodeCheck/main.go [hunk 2]
```diff
@@ -79,21 +79,25 @@ func waitForKuberhealthyEndpointReady(ctx context.Context, kuberhealthyEndpoint
 	doneChan := make(chan error, 1)
 
 	for {
+		res, err := http.Get(kuberhealthyEndpoint)
+		if err != nil {
+			log.Debugln(kuberhealthyEndpoint, "Error making request to kuberhealthy endpoint "+err.Error())
+			continue
+		}
+		if res.StatusCode == 200 {
```
Comment (martinsmatthews) [inline pkg/checks/external/nodeCheck/main.go:87]: possibly we don't need a 200, just getting a response/err being nil is enough?

Comment (elebertus) [reply pkg/checks/external/nodeCheck/main.go:87]: I think technically we could still get a non-200 response and a non-nil error. So checking that we get an healthy response is appropriate still.

**Comment by elebertus** [comment]
I’ll update the go mod bits to try and get the build to pass 👍

**Pull request closed without merging**